asab/web/authz/decorator.py

```diff
@@ -44,15 +44,15 @@ async def wrapper(*args, **kargs):
 				L.error("oauth2_url is not configured ;-(")
```

Comment:
Is `request.AuthzService.OAuth2Url` still the thing?

Reply:
replaced with `_disable_rbac` config switch